gen_pack.sh

```diff
@@ -9,7 +9,7 @@ set -o pipefail
 # Set version of gen pack library
 # For available versions see https://github.com/Open-CMSIS-Pack/gen-pack/tags.
 # Use the tag name without the prefix "v", e.g., 0.7.0
-REQUIRED_GEN_PACK_LIB="0.11.0"
+REQUIRED_GEN_PACK_LIB="0.11.3"
```

Comment:
Currently this causes (known) issue with history generation and must not be merged until we figure out a workaround. You can leave version it as it is 0.11.0 if you with to merge the PR.